tools/firmware-utils/src/xorimage.c

```diff
@@ -24,7 +24,11 @@
 #include <unistd.h>
 #include <sys/stat.h>
 
+#define TRUE 1
```

Comment:
Why not C99's [stdbool.h](https://en.wikipedia.org/wiki/C_data_types#stdbool.h)?

Reply:
I do not have much knowledge about C language.
There are two methods of adding define for TRUE/FALSE like *srec2bin.c* and including ```stdbool.h``` like *mktplinkfw.c*, so I did not know which to use.

Should I use stdbool.h?

Reply:
> Should I use stdbool.h?

I think so, since you are using the "for' loop initial declarations" (a C99 feature) a bit further down in line 118 of the same file.

Reply:
OK, I'll modify it.